src/mumble/GlobalShortcut_unix.cpp

```diff
@@ -131,18 +136,22 @@ GlobalShortcutX::GlobalShortcutX() {
 			 "please enable one of the other methods.");
 	bRunning = true;
 	start(QThread::TimeCriticalPriority);
+#endif  // NO_X11
 }
 
 GlobalShortcutX::~GlobalShortcutX() {
 	bRunning = false;
 	wait();
 
+#ifndef NO_X11
 	if (display)
 		XCloseDisplay(display);
+#endif
 }
 
 // Tight loop polling
 void GlobalShortcutX::run() {
```

Comment:
The name `GlibalShortcutX` seems to imply that this class is exclusively there for handling the shortcuts in X11. Therefore imo it would make more sense to remove the entire class if `NO_X11` is set instead of removing all the function's bodies... :eyes:

Reply:
The class should be split into `GlobalShortcutLinux` and `GlobalShortcutX11` in my opinion.

The `/dev/input/*` method always works, even without a graphical session.

Reply:
Agreed

Reply:
Done, couldn't get `GlobalShortcutLinux` to work for some reason, `evtest` works.